nowcasting_dataset/data_sources/pv/live.py

```diff
@@ -120,7 +125,12 @@ def get_pv_power_from_database(
     pv_yields_df = empty_df.join(pv_yields_df)
     limit = int(interpolate_minutes / 5)
     if limit > 0:
-        pv_yields_df.interpolate(limit=limit, inplace=True)
+        try:
+            pv_yields_df.interpolate(limit=limit, inplace=True, method="cubic")
+        except Exception as e:
```

Comment:
ooh, it's dangerous to catch all exceptions! What happens if `df.interpolate` raises a _different_ exception? That different exception will just get silently swallowed!

I'd suggest we either:

* catch the precise exception subclass we're interested in, or
* add a message to the exception, and re-raise the original exception, like [this](https://stackoverflow.com/a/46091127/732596)

Reply:
ill put it in here https://github.com/openclimatefix/nowcasting_dataset/issues/681